Data-Structures/Linked-List/test/SinglyLinkedList.test.js

```diff
@@ -164,6 +164,32 @@ describe('SinglyLinkedList', () => {
     expect(list.size()).toBe(1)
   })
 
+  it('Middle node of linked list', () => {
+    const list = new LinkedList()
+    list.addFirst(1)
+
+    let MiddleNodeOfLinkedList = list.MiddleOfLL(list.headNode)
+    // Middle node for list having single node
+    expect(MiddleNodeOfLinkedList.data).toEqual(1)
```

Comment:
inline this to `expect(list.findMiddle()).toEqual(1)` (same below); you don't need to pass the `headNode` anymore

Reply:
i think it should be `expect(list.findMiddle().data).toEqual(1)` cause findMiddle returns reference to the middle node rather than just its value.